WordPress/gutenberg
# Snackbar: Use `link` variant for action Button

In preparation for #65751

## What?

Use the `link` variant for the action Button in Snackbar.

## Why?

To reduce custom CSS, and apply the 40px size prop with minimal overrides.

## Testing Instructions

See the "With Action And Explicit Dismiss" story for Snackbar. There should be no visual changes, including focus and hover styles for the action button.
